README.md

```diff
@@ -1,157 +1,160 @@
 
-```javascript
-const monthlyInstallment = totalPayable / numberOfInstallments
-```
+  response
 
-## Challenges to achieve
+  ```javascript
+  {
+    snacks: [
+      {
+        id: string,
+        name: string,
+        price: number,
+      },
+    ];
+  }
+  ```
 
-### CSS
-* **Refactor to make maintainance easier** 
+- POST https://frontend-challenge-vlc-api.public.dev.creditas.io/api/films/confirm
 
-Currently, the project's CSS file has too many styles, making readability difficult. We expect you to organize styles, isolating them in order to make them more readable and easier to maintain.
+  request
 
-### HTML / JS
-* **Add new collateral type _"Home"_** (with its calculation rules) 
-* **Refactor the old code (and arrange the new one) to make maintainance easier** 
+  ```javascript
+  {
+    id: string;
+  }
+  ```
 
-Currently, Creditas offers two products: home equity loan and auto equity loan. This project implements the *"auto equity"* option only, **you must implement the *"home equity"* option**.
-By changing the collateral in the `select` element, the user must see the amount and installment options corresponding to the asset chosen. That is, when selecting either _"Home"_ or _"Auto"_, you must show the correspondent values for each option on the form fields and on the slider.
-Please find the corresponding values below:
+  response
 
-**Auto**
-- Minimum loan amount: R$ 3.000,00;
-- Maximum loan amount: R$ 100.000,00;
-- Number of installments: 24 / 36 / 48 months
-- Collateral minimum value: R$ 5.000,00;
-- Collateral maximum value: R$ 3.000.000,00;
+  ```javascript
+  {
+    seatNumber: number;
+  }
+  ```
 
-**Home**
-- Minimum loan amount: R$ 30.000,00;
-- Maximum loan amount: R$ 4.500.000,00;
-- Number of installments: 120 / 180 / 240 months
-- Collateral minimum value: R$ 50.000,00;
-- Collateral maximum value: R$ 100.000.000,00;
+- POST https://frontend-challenge-vlc-api.public.dev.creditas.io/api/snacks/confirm
 
-Finally, you must update the monthly payment value whenever any input changes.
+  request
 
-### Layout
-The template requires some minor css changes in order to be displayed as proposed in the references provided. Feel free to use any property you feel like to achieve so.
+  ```javascript
+  {
+    id: string;
+  }
+  ```
 
-Desktop version:
-![Desktop version](./desktop-layout.png)
+  response
 
-Mobile version:
-![Mobile version](./mobile-layout.png)
+  ```javascript
+  {
+    queueNumber: number;
+  }
+  ```
 
-### Display the help text
+### Layout
 
-To display the help text, we provide two endpoints on localhost:4000 ( for more details you can check server.js file ):
-- 'api/question' which responses is 
-```
-{ text: 'Question: Can you help me?' }
-```
-- 'api/answer' which response is 
-```
-{ text: 'Answer: Yes sure.' }
-```
+The template requires some css rules in order to be displayed as proposed in the screenshots provided. Feel free to use any property you feel like to achieve so.
 
```

Comment:
Could you also add which items are mandatory to display? Maybe things like `image`, `price` and `title`

Even if it's super obvious for us, worth making it clear to the candidate as well